components/aws/sagemaker/tests/integration_tests/utils/sagemaker_utils.py

```diff
@@ -24,3 +24,32 @@ def describe_hpo_job(client, job_name):
 
 def describe_transform_job(client, job_name):
     return client.describe_transform_job(TransformJobName=job_name)
+
+def describe_workteam(client, workteam_name):
+    return client.describe_workteam(WorkteamName=workteam_name)
+
+def list_workteams(client):
+    return client.list_workteams()
+
+def get_CognitoMemberDefinitions(client):
```

Comment:
nit: Casing on this method doesn't seem to match the others?

Reply:
Fixed.